java-util/src/main/java/io/druid/java/util/common/granularity/NoneGranularity.java

```diff
@@ -25,16 +25,9 @@
 /**
  * NoneGranularity does not bucket data
  */
-public final class NoneGranularity extends Granularity
+public class NoneGranularity extends Granularity
 {
-  private static final NoneGranularity INSTANCE = new NoneGranularity();
-
-  private NoneGranularity() {}
-
-  public static NoneGranularity getInstance()
-  {
-    return INSTANCE;
-  }
+  public NoneGranularity() {}
```

Comment:
I don't think package private is important at all (and as @praveev says it's actually not correct).  

And leaving a comment in this class saying that it is used in `GranularityModule` is kinda weird.  Is it now the responsibility of every class to always comment about all the places that it is being used?  You should be able to do a usages search to figure out where the class is being used.  I'm not really against the comment existing, I just don't see the point in adding it either.